### ISSUE TITLE
checked Status of Menu items written as word by screen reader

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#28860]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

In MuseScore, a menu item such as those in the View menu, e.g. Full screen unchecked F11, Palettes unchecked F9, ... Layout checked F7, ...
The *state* "checked or unchecked" is written out as a word, as part of the *name* of the item. So on a braille display, you can read the whole word "checked".

Compare this with many other Windows programs, e.g. Notepad:
(Windows 10, Format, Word Wrap)
(Windows 11, View, Word Wrap)
Notice that the unchecked state is not shown at all, and if you toggle it on, a checkmark *state* appears rather than the word "checked".
It is shown on the braille display as the symbol for a checked checkbox.
So, the menu item is given a checked status, rather than changing the name of the menu item to include the word "checked".

You will probably only notice this problem if you use a braille display, but it is pretty obvious when you do.

### Supporting files, videos and screenshots

N/A

### What is the latest version of MuseScore Studio where this issue is present?

4.5

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
